library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.java

```diff
@@ -269,20 +253,38 @@ public void deactivate()
     store.disconnect();
   }
 
+  protected void insertResultsInQueue(PreparedStatement preparedStatement) throws SQLException, InterruptedException
```

Comment:
Please suggest a better name. The only contribution of this function is to insert rows/results in emit queue.